core/state/metrics.go

```diff
@@ -19,12 +19,10 @@ package state
 import "github.com/ethereum/go-ethereum/metrics"
 
 var (
-	accountUpdatedMeter      = metrics.NewRegisteredMeter("state/update/account", nil)
-	storageUpdatedMeter      = metrics.NewRegisteredMeter("state/update/storage", nil)
-	accountDeletedMeter      = metrics.NewRegisteredMeter("state/delete/account", nil)
-	storageDeletedMeter      = metrics.NewRegisteredMeter("state/delete/storage", nil)
-	accountTrieUpdatedMeter  = metrics.NewRegisteredMeter("state/update/accountnodes", nil)
-	storageTriesUpdatedMeter = metrics.NewRegisteredMeter("state/update/storagenodes", nil)
-	accountTrieDeletedMeter  = metrics.NewRegisteredMeter("state/delete/accountnodes", nil)
-	storageTriesDeletedMeter = metrics.NewRegisteredMeter("state/delete/storagenodes", nil)
+	accountUpdatedMeter = metrics.NewRegisteredMeter("state/update/account", nil)
+	storageUpdatedMeter = metrics.NewRegisteredMeter("state/update/storage", nil)
+	accountDeletedMeter = metrics.NewRegisteredMeter("state/delete/account", nil)
+	storageDeletedMeter = metrics.NewRegisteredMeter("state/delete/storage", nil)
+	accountTrieMeter    = metrics.NewRegisteredMeter("state/trie/account", nil)
+	storageTriesMeter   = metrics.NewRegisteredMeter("state/trie/storage", nil)
```

Comment:
Why remove the delete/update pair? And what does `state/trie/account` mean now? Is it the updates? If so, we should make the `update` part of the path. But I really think we need to retain delete, otherwise it's a bit of a moot?

Reply:
OK, I will change it back